app/Models/Device.php

```diff
@@ -35,7 +35,7 @@ class Device extends BaseModel
 
     public $timestamps = false;
     protected $primaryKey = 'device_id';
-    protected $fillable = ['hostname', 'ip', 'status', 'status_reason', 'sysName', 'sysDescr', 'sysObjectID', 'hardware', 'version', 'features', 'serial', 'icon'];
+    protected $fillable = ['hostname', 'ip', 'status', 'status_reason', 'sysName', 'sysDescr', 'sysObjectID', 'hardware', 'version', 'features', 'serial', 'icon', 'overwrite_ip', 'os', 'community', 'port', 'transport', 'snmpver', 'poller_group', 'port_association_mode', 'snmp_disable'];
```

Comment:
Missing the snmpv3 fileds.
Check against the array in createHost()
Also, you can make this array vertical (one item per line)